src/services/postDetail.ts

```
@@ -0,0 +1,454 @@
+        mutation: PostDetailGQL.MUTATIONS.DELETE_BOARDING_ROOM_COMMENT,
+        variables: { commentId }
+      });
+      return data?.deleteBoardingRoomComment || '';
+    } catch (error) {
+      console.error('하숙방 댓글 삭제 오류:', error);
+      throw error;
+    }
+  },
+
+  // 일반 게시물 수정/삭제 (하숙방은 수정 불가)
+  updatePost: async (client: ApolloClient<any>, postUpdateInput: any): Promise<string> => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)